Src/World/Timing.h

```diff
@@ -5,38 +5,46 @@
 
 class Timing {
     private:
-        double timeStep;
-        double accumulatedSeconds;
-
-        double secondToLastFixedTime;
-        double lastFixedTime;
-
-        // Total time since the object's initialization.
         std::chrono::high_resolution_clock::time_point initialTime;
-        // Previous time when the last call to getElapsedSeconds() was made.
         std::chrono::high_resolution_clock::time_point prevTime;
 
-    public:
-        Timing(int tickrate);
-
-        double getTimeStep() const;
-        void addSecondsToAccumulator(double secondsPassed);
-
-        // Returns whether enough time is left on the accumulator for another tick.
-        bool tickReady() const;
+        // Tick timers (Milliseconds)
+        int tickRate;
+        int tickStep;
+        float sinceLastTick;
+        float avgTickRate;
```

Comment:
The tick rate is supposed to be fixed. What purpose does this serve?

Reply:
Because there is no guarantee of how long a tick will or is taking, against whether or not the program has slowed down for outside reasons.
I have noticed that the true tick rate drops to about half when the game is not a focused window.